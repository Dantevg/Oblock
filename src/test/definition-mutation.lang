{
	var z
	x := -5
	a := {x := x + 1}
	if x < 10: {
		x = -x
		z = 1
	}
	print x
	print a.x
	print z
}
