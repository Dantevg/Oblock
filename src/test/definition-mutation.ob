{
	var z
	x = -5
	a = { var x = x + 1 }
	if x < 10: {
		x = -x
		z = 1
	}
	print x
	print(a.x)
	print z
	
	b = { 37 + 5 = "forty-two" }
	"hello world" = "hehe"
	print(b.42, ."hello world")
}
