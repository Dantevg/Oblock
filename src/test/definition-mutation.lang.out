5
-4
1
forty-two	hehe
